jme3-core/src/main/java/com/jme3/export/SavableClassUtil.java

```diff
@@ -234,14 +233,11 @@ public static Savable fromName(String className, List<ClassLoader> loaders) thro
     private static Constructor findNoArgConstructor(String className)
             throws ClassNotFoundException {
         Class clazz = Class.forName(className);
-        Constructor[] allConstructors = clazz.getDeclaredConstructors();
-
-        Constructor result = null;
-        for (Constructor constructor : allConstructors) {
-            if (constructor.getParameterTypes().length == 0) {
-                result = constructor;
-                break;
-            }
+        Constructor result;
+        try {
+            result = clazz.getDeclaredConstructor();
+        } catch (NoSuchMethodException e) {
+            result = null;
```

Comment:
Sorry @stephengold, I mean instead of returning null result you can just throw 

```
throw new InstantiationException(
                    "Loading requires a no-arg constructor, but class "
                    + className + " lacks one.");
```

inside catch block.

Reply:
Done.

Reply:
Thanks :)